docs/resources/Guild.md

```diff
@@ -831,12 +831,12 @@ Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object)
-| channel_id                   | snowflake           | id of channel to move user to (if they are connected to voice)                                                                                                                                                                           | MOVE_MEMBERS     |
-| communication_disabled_until | ?ISO8601 timestamp  | when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout | MODERATE_MEMBERS |
+| nick                         | string              | value to set user's nickname to                                                                                                                                                                                                                                                            | MANAGE_NICKNAMES |
+| roles                        | array of snowflakes | array of role ids the member is assigned                                                                                                                                                                                                                                                  | MANAGE_ROLES     |
+| mute                         | boolean             | whether the user is muted in voice channels. Will throw a 400 if the user is not in a voice channel                                                                                                                                                                                        | MUTE_MEMBERS     |
+| deaf                         | boolean             | whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel                                                                                                                                                                                    | DEAFEN_MEMBERS   |
+| channel_id                   | snowflake           | id of channel to move user to (if they are connected to voice)                                                                                                                                                                                                                            | MOVE_MEMBERS     |
+| communication_disabled_until | ?ISO8601 timestamp  | when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout. Will throw a 403 if the user has administrator | MODERATE_MEMBERS |
```

Comment:
```suggestion
| communication_disabled_until | ?ISO8601 timestamp  | when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout; it will throw a 403 error if the user has administrator permisson | MODERATE_MEMBERS |
```

Reply:
Again, as i said above this just makes it inconsistent with the rest of the table

Reply:
i see, are you saying about the `;`?
because it's not all that i suggested, for example, what is a "403"? i think it doesn't make sense to some people without the word "error"